Mathlib/Algebra/Homology/HomologicalComplex.lean

```diff
@@ -732,37 +733,25 @@ end OfHom
 
 section Mk
 
--- porting note: removed @[nolint has_nonempty_instance]
-/-- Auxiliary structure for setting up the recursion in `mk`.
-This is purely an implementation detail: for some reason just using the dependent 6-tuple directly
-results in `mk_aux` taking much longer (well over the `-T100000` limit) to elaborate.
--/
-structure MkStruct where
-  (X₀ X₁ X₂ : V)
-  d₀ : X₁ ⟶ X₀
-  d₁ : X₂ ⟶ X₁
-  s : d₁ ≫ d₀ = 0
-#align chain_complex.mk_struct ChainComplex.MkStruct
-
 variable {V}
 
 /-- Flatten to a tuple. -/
-def MkStruct.flat (t : MkStruct V) :
+def _root_.CategoryTheory.ShortComplex.flat (t : ShortComplex V) :
```

Comment:
I know it is orthogonal to this speed-up, but would you mind renaming `flat` to something like `asTuple`? Because `ShortComplex.flat` gives me all kinds of "commutative algebra flatness" vibes that have nothing todo with this defn.

Reply:
Done